### PR TITLE
Implement cleanAuxiliary and cleanArtifacts commands

### LINKED ISSUE #11: Implement server commands to clean build directory
As of texlab v4.1.0, https://github.com/latex-lsp/texlab/issues/607

Just a log here. I am not going to do this but maybe someone will.

### PR DESCRIPTION
**Pull request by LDAP**
This PR adds the commands described here: https://github.com/latex-lsp/texlab/issues/607.

Closes #11

### CODE REVIEW

**Comment by jfcherng** [comment]
Hi, @LDAP I don't use texlab honestly so if you think this PR is okay and no one else wants to review this, feel free to merge it.

File: plugin.py [hunk 1]
```diff
@@ -144,3 +145,35 @@ def on_response_async(self, response: Any) -> None:
...
+    def run(self, _: sublime.Edit) -> None:
+        session = self.session_by_name(PLUGIN_NAME)
+        if not session:
+            return
+
+        execute_command_params = {
+            "command": "texlab.cleanArtifacts",
+            "arguments": [text_document_identifier(self.view)]
+        }  # type: ExecuteCommandParams
+        session.execute_command(execute_command_params, False)
```
Comment (rchl) [inline plugin.py:179]: I believe that those can be handled by the built-in `lsp_execute` command - https://lsp.sublimetext.io/commands/#execute-server-commands

Comment (LDAP) [reply plugin.py]: That is a great solution. Did not know about `lsp_execute`. Thanks a lot for your input!

File: plugin.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+from LSP.plugin.core.protocol import ExecuteCommandParams
```
Comment (rchl) [inline plugin.py:1]: I know this is merged already but this is unused now, I guess. You can remove it at your discretion :)

**Pull request merged**